recipes-core/images/aos-image.inc

```diff
@@ -67,7 +67,7 @@ IMAGE_INSTALL:append = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'selinux', ' \
         packagegroup-selinux-minimal \
         packagegroup-selinux-policycoreutils \
-        aos-selinux-autorelabel \
+        selinux-autorelabel \
```

Comment:
Is it a standard one? We don't need custom autorelabel service anymore. Do we?

Reply:
Unfortunately, we need it because necessary to auto-relable some of the directories (var, home etc.)